### DISCUSSION TITLE
How long will github reply a ticket? #2510045

### DISCUSSION THREAD

**Discussion by 0vid1u** [community/community#85318] [New to GitHub]
Was billed twice for the same period using the Copilot subscription and my ticket to support on fixing that is staying in open status more then a week. I am interested if there are other people in the same situation and if it is somehow possible to reach them faster?

**Comment by akashdip2001** [comment]
If you're facing billing issues with GitHub Copilot and your support ticket has been open for an extended period, consider the following steps:

1. **Check GitHub Community Forums:**
   - Visit GitHub's Community Forums to see if others have experienced similar billing problems.
   - Users might share their experiences and solutions there.

2. **Social Media:**
   - Check GitHub's official social media accounts (e.g., Twitter) for any updates or announcements regarding billing issues.
   - Sometimes companies respond faster to inquiries made through social media platforms.

3. **Escalate the Support Ticket:**
   - If your ticket has been open for an extended period without resolution, try updating it with a polite request for escalation or follow-up.
   - Mention the urgency and the extended wait time.

4. **Contact GitHub Sales or Support Directly:**
   - Reach out to GitHub Sales or Support directly through their official contact channels.
   - Provide them with your ticket number and a brief summary of your issue.

Remember to keep your communication polite and include relevant details to facilitate a quicker resolution. If the issue persists, reaching out through various channels can sometimes expedite the process.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²
